docs/changelog.rst

```diff
@@ -4,7 +4,12 @@
 Changelog
 =========
 
+- :breaking:`208` Split ``fakeredis`` optional dependency from the ``async-rediscache`` extra. You can now install with ``[fakeredis]`` to just install fakeredis (with lua support), ``[async-rediscache]`` to install just ``async-rediscache``, or use either ``[all]`` or ``[async-rediscache,fakeredis]`` to install both.
```

Comment:
I think people tend to jump on `breaking` changes more than other ones, which makes me wonder whether it's a good idea to mention that we're splitting them in order to allow for python support here.